build/update-crdb-certs.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+if [ "$1" == "" ]; then
+  echo "usage: ./apply-certs.sh <CLUSTER_CONTEXT> <NAMESPACE>"
+  exit 1
+fi
+
+if [ "$2" == "" ]; then
+  echo "usage: ./apply-certs.sh <CLUSTER_CONTEXT> <NAMESPACE>"
+  exit 1
+fi
+
+echo '========================================================================='
+echo '= Note that errors below are acceptable as long as the terminal message ='
+echo '= is success.                                                           ='
+echo '========================================================================='
+
+set -e
+set -x
+
+# Paths to directories in which to store certificates and generated YAML files.
+CONTEXT="$1"
+DIR="$(pwd)"
+NAMESPACE="$2"
+CA_CRT_DIR="$DIR/workspace/$CONTEXT/ca_certs_dir"
```

Comment:
This seems to imply a requirement that this script be run from the `build` as the working directory.  Can we document that and/or validate it programatically?